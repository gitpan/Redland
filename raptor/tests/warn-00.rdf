<?xml version="1.0" standalone="no"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
         xmlns:ex="http://example.org/ns#">

  <!-- Should give a warning about non-prefixed element but not crash -->

  <rdf:Description rdf:about="http://example.org/node">
    <ex:prop><el/></ex:prop>
  </rdf:Description>

</rdf:RDF>
