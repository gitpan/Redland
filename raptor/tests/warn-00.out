<http://example.org/node> <http://example.org/ns#prop> "" .
